tests/kafkatest/services/console_consumer.py

```diff
@@ -209,7 +227,13 @@ def _worker(self, idx, node):
         # Create and upload config file
         self.logger.info("console_consumer.properties:")
 
-        prop_file = self.prop_file(node)
+        if self.client_prop_file_override:
+            prop_file = self.client_prop_file_override
+            self.security_config = self.kafka.security_config.client_config(prop_file, node, self.jaas_override_variables)
```

Comment:
this can be just ```self.kafka.security_config.client_config(prop_file, node)```?

Reply:
Many thanks. @omkreddy can you take a look at the new patch?